cajita/unit_test/tstLocalGrid.hpp

```diff
@@ -1830,6 +1930,106 @@ void notPeriodicTest3d()
                 }
             }
 
+    // Check the boundary spaces.
+    IndexSpace<3> boundary_cell_space;
+    if ( -1 == local_grid->neighborRank( -1, 0, 1 ) )
+    {
+        boundary_cell_space =
+            local_grid->boundaryIndexSpace( Cell(), -1, 0, 1 );
+        EXPECT_EQ( boundary_cell_space.min( Dim::I ), 0 );
```

Comment:
by the definition of `sharedIndexSpaceImpl` of off_ijk = -1,  owned_space.min( d ) - halo_width  = 0